integrations/dynatrace/.port/resources/blueprints.json

```diff
@@ -0,0 +1,264 @@
+          "enum": ["CLOSED", "OPEN"],
+          "enumColors": {
+            "CLOSED": "green",
+            "OPEN": "red"
+          }
+        },
+        "startTime": {
+          "type": "string",
+          "title": "Start Time",
+          "description": "The start time of the problem, in UTC milliseconds.",
+          "format": "date-time"
+        },
+        "endTime": {
+          "type": "string",
+          "title": "End Time",
+          "description": "The end time of the problem, in UTC milliseconds.",
+          "format": "date-time"
+        }
+      },
+      "required": []
+    },
+    "relations": {
+      "impactedEntities": {
+        "title": "Impacted Entities",
+        "target": "dynatraceEntity",
+        "required": false,
+        "many": true
+      },
+      "linkedProblemInfo": {
+        "title": "Linked Problem Info",
+        "target": "dynatraceProblem",
+        "required": false,
+        "many": false
+      },
+      "rootCauseEntity": {
+        "title": "Root Cause Entity",
+        "target": "dynatraceEntity",
+        "required": false,
+        "many": false
+      }
+    },
+    "mirrorProperties": {},
+    "calculationProperties": {}
+  },
+  {
+    "identifier": "dynatraceEntity",
+    "description": "This blueprint represents a Dynatrace Entity",
+    "title": "Dynatrace Entity",
+    "icon": "Dynatrace",
+    "schema": {
+      "properties": {
+        "firstSeen": {
+          "type": "string",
+          "title": "First Seen",
+          "description": "The timestamp at which the entity was first seen, in UTC milliseconds.",
+          "format": "date-time"
+        },
+        "lastSeen": {
+          "type": "string",
+          "title": "Last Seen",
+          "description": "The timestamp at which the entity was last seen, in UTC milliseconds.",
+          "format": "date-time"
+        },
+        "type": {
+          "type": "string",
+          "title": "Type",
+          "description": "The type of the entity."
+        },
+        "tags": {
+          "type": "array",
+          "title": "Tags",
+          "description": "A list of tags of the entity."
+        },
+        "managementZones": {
+          "type": "array",
+          "title": "Management Zones",
+          "description": "A list of all management zones that the entity belongs to."
+        },
+        "properties": {
+          "type": "properties",
+          "title": "Properties",
+          "description": "A list of additional properties of the entity."
+        },
+        "fromRelationships": {
+          "type": "object",
+          "title": "From Relationships",
+          "description": "A list of all relationships where the entity is the source."
+        },
+        "toRelationships": {
+          "type": "object",
+          "title": "To Relationships",
+          "description": "A list of all relationships where the entity is the target."
+        }
+      }
```

Comment:
shouldn't this be a relation?

Reply:
Not exactly, both are ambiguous objects with little to not information given on the contents. Since the schema isn't standardized, we can't reliably pull a relationship out of it. I'll rather remove it.